### PR TITLE
doc: add meeting notes 2023-02-06

### PR DESCRIPTION
**Pull request by UlisesGascon**
Signed-off-by: Ulises Gascon <UlisesGascon@users.noreply.github.com>

Co-authored-by: Yagiz Nizipli <yagiz@nizipli.com>
Co-authored-by: Marvin Hagemeister <hello@marvinh.dev>

### CODE REVIEW

**Review by anonrig — APPROVED** [review]

**Comment by anonrig** [comment]
@nodejs/performance

File: meetings/2023-02-06.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+19.x changes are already merged
+There is an ongoing work to port the changes to 16.x and 18.x (lead by @joyeecheung)
+This item will be removed from the agenda
+### nodejs/performance
+
+* AbortSignal is slow [#44](https://github.com/nodejs/performance/issues/44)
+Reached out to RxJS folks. There are some discussions and agreements ongoing, but after checking the performance improvements there were no major upgrades
+PR from Santigo (@santigimeno) improves the performance by 20% - https://github.com/nodejs/node/pull/46527 
+Ongoing discussion about if this can break the spec or not
+Recommendation escalate to WinterCG group
```
Comment (santigimeno) [inline meetings/2023-02-06.md:42]:
```suggestion
Ongoing discussion about further changes that could improve the performance even more include moving `Event.isTarget` to the prototype and not setting `__proto__: null`. The former might break the spec. 
Recommendation escalate to WinterCG group
```

**Review by santigimeno — APPROVED** [review]
LGTM with comments

File: meetings/2023-02-06.md (hunk 1 above)
Comment (UlisesGascon) [reply meetings/2023-02-06.md]: thanks @santigimeno ! :)

File: meetings/2023-02-06.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+* Rafael Gonzaga: @RafaelGSS
+* Robert Nagy: @ronag
+
+
+## Agenda
+
+## Announcements
+
+[Ada](https://github.com/ada-url/ada), the new URL parse will be added to Node.js as dependency in order to increase speed by 78 %, fetch will become faster too 🎉
+Marvin released a really good blog post on https://marvinh.dev/blog/speeding-up-javascript-ecosystem-part-3/.
```
Comment (tniessen) [inline meetings/2023-02-06.md:26]: If these two are unrelated, they should not be on the same line.

```suggestion
[Ada](https://github.com/ada-url/ada), the new URL parse will be added to Node.js as dependency in order to increase speed by 78 %, fetch will become faster too 🎉

Marvin released a really good blog post on https://marvinh.dev/blog/speeding-up-javascript-ecosystem-part-3/. 
```

Comment (UlisesGascon) [reply meetings/2023-02-06.md]: thanks @tniessen  👍

**Review by tony-go — APPROVED** [review]
